source/documentation/runbooks/aws-credentials.html.md.erb

```diff
@@ -0,0 +1,33 @@
+---
+title: AWS credentials remediation process
+last_reviewed_on: 2021-11-16
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+These are the steps to be followed if we learn a set of AWS credentials have been compromised. 
```

Comment:
Is it worth adding a note to say that keys exposed on Cloud Platform are managed by the Cloud Platform Team. I wouldn't expect Ops Eng to deal with those types of security incidents?

Reply:
Done :)